terraform/services/github-actions-runner/main.tf

```diff
@@ -80,7 +80,7 @@ module "github-actions-runner" {
   role_permissions_boundary = data.aws_iam_policy.developer_boundary_policy.arn
 
   enable_ssm_on_runners = true
-  enable_userdata       = false
+  enable_userdata       = true
```

Comment:
Revert of userdata to true should be in another PR. If anything this PR should switch this back to false.